js/content-scripts/tvdb.js

```diff
@@ -0,0 +1,57 @@
+let overseerrContainer, imdbId, tmdbId, mediaType, mediaInfo;
+
+containerOptions.anchorElement = '#translations';
+containerOptions.containerClass = 'mb-2 py-2';
+containerOptions.badgeBackground = '#313131';
+
+const imdbRegex = /\/title\/(tt\d+)(?:\/|$).*/;
+
+mediaType = document.location.pathname.startsWith('/movies') ? 'movie' : 'tv';
+let matches = $("a[href*='/title/tt']", $('#series_basic_info')).attr( "href" ).match(imdbRegex);
+// console.log(matches);
+if (matches !== null && matches.length > 1) {
+    imdbId = matches[1];
+    console.log(`IMDb id: ${imdbId}`);
+
+    let title = $('#series_title').text();
+    // let releaseYear = parseInt($('a.TitleBlockMetaData__StyledTextLink-sc-12ein40-1.rgaOW:first').text());
+
+    initializeContainer();
+    insertSpinner();
+
+    pullStoredData(function () {
+        if (!userId) {
+            removeSpinner();
+            insertNotLoggedInButton();
+            return;
+        }
+
+        chrome.runtime.sendMessage({contentScriptQuery: 'search', title: title}, json => {
```

Comment:
This call can be avoided, and should be since we can retrieve the TMDB id directly, see previous comment.